### ISSUE TITLE
Simulator micro:bit logo doesn't have focus indicator

### ISSUE THREAD

**Issue by microbit-matt-hillsdon** [microsoft/pxt-microbit#4484]
**Describe the bug**
The micro:bit logo on the simulator doesn't have a visible focus indicator when you tab to it. 
This makes it hard for keyboard users to understand when it is selected.

**To Reproduce**
1. Add "on logo pressed" block with a "show string" so you can tell it triggers
2. Attempt to focus the touch logo with the keyboard and trigger the "on logo pressed" with Enter
3. Note that it is possible, but there is no focus indicator so it's hard to tell when it is focussed

The outline used for the A/B buttons doesn't apply here as the DOM structure is different (same element is .sim-button-outer .sim-button-group). Outlines in general have been disabled via CSS.

**Expected behavior**
An outline similar to the A/B buttons.

**Screenshots**
![image](https://user-images.githubusercontent.com/44397098/157227878-e1a67006-d3ca-48e3-8ac8-d7edc6caa693.png)

**micro:bit version (please complete the following information):**

Sim only.

**Desktop (please complete the following information):**
 - OS: Mac 11.6.1 (20G224)
 - Browser: Google Chrome	
 - Version: 99.0.4844.51 (Official Build) (x86_64)

**Comment by kimprice** [comment]
Reopening - This bug is fixed on Edge, but we are seeing this bug on Google Chrome.

**Comment by kimprice** [comment]
Actually, it is being focused just fine when the simulator shown is a V2 board. The problem is that V1 boards in the simulator should not show the logo as clickable. New issue filed: https://github.com/microsoft/pxt-microbit/issues/4723
